tests/pyautomagic/test_performEOGRegression.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+import numpy as np
+import logging
+
+def performEOGRegression(eeg, eog, *args):
```

Comment:
It's unnecessary to copy the same function here, you can simply import EOGRegression